mitxgraders/helpers/calc/expressions.py

```diff
@@ -996,10 +1020,10 @@ def eval_parallel(parse_result):
         >>> MathExpression.eval_parallel([1,1])
         0.5
         >>> MathExpression.eval_parallel([1,0])
-        nan
+        0
```

Comment:
Now need to update the docstring to match this behavior.

Reply:
Line 1008, I think?

Reply:
done